evaluation/cpps/verification.py

```diff
@@ -21,8 +21,8 @@ def evaluate_database(path):
             [[mf_count]] = db.fetch_all(
                 "SELECT COUNT(DISTINCT MaterialFlowOrderId) FROM MaterialFlowOrderHistory")
             for i in range(1, mf_count+1):
-                sql_query = "SELECT Timestamp_ms, State FROM MaterialFlowOrderHistory WHERE MaterialFlowOrderId = " + \
-                    str(i) + " ORDER BY Timestamp_ms"
+                sql_query = "SELECT Timestamp_ms/10 as t, State FROM MaterialFlowOrderHistory WHERE MaterialFlowOrderId = " + \
+                    str(i) + " ORDER BY t"
```

Comment:
I had to add this because of the "wrong" ordering of order status updates in the database. When this other issue is fixed, this line can be reverted again.